models/maverick/maverick_trades.sql

```diff
@@ -0,0 +1,48 @@
+{{ config(
+    schema = 'maverick',
+    alias = 'trades',
+    post_hook='{{ expose_spells(\'["arbitrum", "ethereum", "optimism", "polygon"]\',
```

Comment:
For now may be just ethereum?
```suggestion
    post_hook='{{ expose_spells(\'["ethereum"]\',
```

Reply:
I think this is fixed.  It isn't letting me commit this suggestion because it is already there.

@Hosuke